src/app/file-manager/transferables-grid/transferables-grid.component.ts

```diff
@@ -104,49 +106,51 @@ export class TransferablesGridComponent implements OnInit, AfterViewInit {
 
```

Comment:
Many of the fields above here appear to be unused: `selectedRows`, `idCounter`, `pageSize`, `pageSizeOptions`, `pageEvent`. If they're unused they should be removed.